tests/end_to_end/test_runtimes_e2e.py

```diff
@@ -0,0 +1,106 @@
+import json
+from unittest import TestCase
+
+import os
+
+from pathlib import Path
+
+from parameterized import parameterized_class
+
+from tests.end_to_end.end_to_end_base import EndToEndBase, EndToEndTestContext
+from tests.end_to_end.test_stages import (
+    DefaultInitStage,
+    DefaultRemoteInvokeStage,
+    DefaultDeleteStage,
+    EndToEndBaseStage,
+    DefaultSyncStage,
+)
+from tests.testing_utils import CommandResult
+
+
+class HelloWorldValidators(TestCase):
```

Comment:
nit: These are not really `HelloWorldValidators` though, they are more generic than that.